pkg/scheduler/objects/application_events.go

```diff
@@ -20,6 +20,7 @@ package objects
 
 import (
 	"fmt"
+	"github.com/apache/yunikorn-core/pkg/common/resources"
```

Comment:
linter

Reply:
Thank you @pbacsko , fixed in latest PR.